.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,56 @@
+name: Unit tests
+on: [push, pull_request]
+
+env:
+  SCOREP_TIMER: clock_gettime # tsc causes warnings
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python: [2.7, 3.5, 3.6, 3.7, 3.8]
+      fail-fast: false
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip
+    - id: cache
+      uses: actions/cache@v1
+      with:
+        path: ~/scorep
+        key: scorep
+
+    - name: Install system dependencies
+      run: sudo apt install gcc-7-plugin-dev libopenmpi-dev openmpi-bin
+    - name: Install dependencies
+      if: steps.cache.outputs.cache-hit != 'true'
+      run: |
```

Comment:
Is the Score-P build Cached? Otherwise it might be rasonable to use the ubuntu package her.

Reply:
Yes, see the line above: `      if: steps.cache.outputs.cache-hit != 'true'`